### DISCUSSION TITLE
Polling the mainling of Nonperishable Overhaul's Changes

### DISCUSSION THREAD

**Discussion by RobbieNeko** [cataclysmbn/Cataclysm-BN#4115] [Polls]
As noted by Royal Fox on [the PR](https://github.com/cataclysmbnteam/Cataclysm-BN/pull/4113), this likely deserves a poll.

The changes made by Nonperishable Overhaul are, primarily, making certain items that either already had long perish times anyway or don't really make sense to perish (at least not in a timescale that you'd see in-game) completely nonperishable.

As such:

**Comment by Relgar** [comment]
Which consumables do these affect exactly? What will change?

**Reply by RobbieNeko** [reply]
The primary examples are the various nuts, salted and dried meats, dried fruits and veggies, processed fat (tallow and cracklings), some of the teas that still had a time until they spoiled for some reason, and a bunch of the junk food. So, as stated, mostly a bunch of stuff that either already had very long spoilage times or that conceptually made sense to not spoil. (Though I very much so don't claim to have gotten all of the items that fit those categories considering I just followed the list in the original mod)
